tailwindcss-uswds/tokens/components.json

```diff
@@ -8756,23 +8512,16 @@
     ".usa-nav__close": {
         "MozOsxFontSmoothing": "inherit",
         "WebkitFontSmoothing": "inherit",
-        "color": [
-            "#005ea2",
-            "currentColor"
-        ],
+        "color": ["#005ea2", "currentColor"],
         "textDecoration": "underline",
         "backgroundColor": "transparent",
         "border": "0",
         "borderRadius": "0",
         "boxShadow": "none",
+        "fontWeight": "normal",
+        "margin": ["0", "-0.75rem -1rem 1rem auto"],
+        "padding": "0",
         "textAlign": ["left", "center"],
-        "cssFloat": "right",
```

Comment:
Why did you remove these styles?

Reply:
I took a look through my copy of `components.json` and compared it to the original repo's `components.json` and I think all of the original styles are intact. Do you know what class that was being applied to? It may just be a difference in the formatting of the code; I have it set to auto-format on save on my machine.

Reply:
Sorry, just now seeing what you meant; I have no idea why it was so hard for me to find this, haha. Nor do I know why those styles were deleted in the first place. It may have happened when I tried to sync after the changes or I just accidentally deleted them somehow. I confirmed those styles are in place in my current copy, though.